src/Geometry.cpp

```diff
@@ -1081,6 +1105,10 @@ bool Geometry::withinBounds(LocalCoords* coords){
 }
 
 
+void Geometry::setFSRCentroid(int fsr, Point* centroid){
```

Comment:
This method needs as Doxygen comment block. And since this is a public method (which I think is dangerous, but I understand why), it should have a ``@details`` block which describes it specifically as a helper routine for the ``TrackGenerator`` and not as something for the user to call from Python.

Speaking of public vs. private methods, we could use the SWIG ``%ignore`` directive to enforce that routines like this one are not exposed within Python. What do you think about this @samuelshaner? This might also apply to the similar discussion we've had with @geogunow about a few questionably public routines needed for the ``TrackGenerator`` and ``Geometry`` to communicate with one another during ray tracing.